graphqlapi/src/main/java/io/stargate/graphql/web/resources/schemafirst/SchemaFirstCache.java

```diff
@@ -74,12 +76,13 @@ public GraphQL getAdminGraphql() {
     return adminGraphql;
   }
 
-  public GraphQL getGraphql(String namespace, Map<String, String> headers) throws Exception {
-    return getGraphql(persistence.decorateKeyspaceName(namespace, headers));
+  public GraphQL getGraphql(String namespace, Map<String, String> headers, Optional<UUID> version)
+      throws Exception {
+    return getGraphql(persistence.decorateKeyspaceName(namespace, headers), version);
   }
 
-  private GraphQL getGraphql(String namespace) throws Exception {
-    SchemaSource latestSource = schemaSourceDao.getLatest(namespace);
+  private GraphQL getGraphql(String namespace, Optional<UUID> version) throws Exception {
+    SchemaSource latestSource = schemaSourceDao.getByVersion(namespace, version);
```

Comment:
One solution for exposing the schema to the end-user would be to rework this method to return the `GraphqlHolder`. The caller would be able to access the `SchemaSrouce` but also `GraphQL`. 
However, in the `SchemaSource` java-doc I see that:
```
 * <p>Usually this schema is partial, it will be completed (with default/generated types, directive
 * definitions...) to produce the actual one used at runtime.
```
so I am not sure if this is proper information that should be returned to a client.